TodoTests/XTests.swift

```diff
@@ -11,10 +11,16 @@ import XCTest
 @testable import Todo //DONT FORGET
 
 
+
 class XTests: XCTestCase {
     
-    func testThatSomething() {
+    let td = TodoViewModel()
+    
+    func testTransformedData() {
+        
+        let s = td.transformedDatasource("Jerry")
         
+        XCTAssert(s[0] == "Jerry:")
```

Comment:
Dope!

Looking at this test I can quickly understand your decision making around modifying the `transformedDatasource` function with the `string` param.  (A side note, be cautious about naming variables with reserved keywords such as `string`.  I usually will tack a prefix on the variable name, so I'd maybe call it `stringToTransform` or something like that.  (`stringToTransform` is a pretty shitty name but you catch my drift.

I think that this test also shows the need for the next step in our design - we need a way to input or plug data into the actual viewModel!  Right now it's hard-coded - so we'll do some refactoring there.  I'll make a task for that and we can discuss.

As for the test, you are checking `s[0]` which is good, but how would you modify this test to test that every element in `s` is correct?